codalab/server/cookie.py

```diff
@@ -11,6 +11,45 @@
 
 from codalab.server.authenticated_plugin import user_is_authenticated
 
+class NetCatCookie(object):
+    """
+    Represents the user's session cookie after logging in.
```

Comment:
Explain why the cookie is necessary?

Reply:
not necessary, just a leftover remnant from previous experiments